docs(ECO-2911): Add a `README.md` to the SDK (#646)
econia-labs/emojicoin-dot-fun: src/typescript/sdk/README.md

# @econia-labs/emojicoin-sdk

## Overview

The emojicoin SDK was originally used to supplement the next.js frontend

> with server-side queries and API keys. As such, there are a few setup

Author: ```suggestion
with server-side queries that use private API keys. As such, there are a few setup
```

> with server-side queries and API keys. As such, there are a few setup

Author: ```suggestion
with server-side queries that use private API keys. As such, there are a few
```

> configurations and quirks to be mindful of when using the SDK.

Author: ```suggestion
configuration options and quirks to be mindful of when using the SDK.
```
